languages/javascript.mdx

```diff
@@ -1,19 +1,88 @@
 ---
-title: 'JavaScript'
-icon: 'square-js'
+title: "JavaScript"
+icon: "square-js"
 ---
 
+To get started, create an account by clicking “_Log in_” on [PromptLayer](https://promptlayer.com/). Once logged in, click the button to create an API key and save this in a secure location ([Guide to Using Env Vars](https://nodejs.dev/en/learn/how-to-read-environment-variables-from-nodejs/)).
 
-<Warning> Javascript support is experimental and will not be updated as frequently as the Python one. Follow [@promptlayer](https://twitter.com/promptlayer) for updates and check back here for Javascript requirements.</Warning>
```

Comment:
Our javascript library is not finished so keep a warning here.

<Warning>Javascript support is still in development. The Javascript library is not yet feature-complete. Check back here for updates.</Warning>

Reply:
Reverted.